modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -647,6 +647,7 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
     virtual Ptr<BackendNode> initVkCom(const std::vector<Ptr<BackendWrapper> > &inputs) CV_OVERRIDE
     {
 #ifdef HAVE_VULKAN
+        CV_Assert(!blobs.empty());
```

Comment:
I think we need this assert in other init functions as well: `initWebnn`, `initNgraph`, `initHalide`.